docs/cli/serve.md

```diff
@@ -0,0 +1,24 @@
+# Serving a QueryBot: Llamabot CLI Guide
```

Comment:
## Intent

The intent of the changes in the file is to introduce and explain how to use the `serve` command of the Llamabot CLI, which enables users to create an Ollama compatible server for their querybot.

## Summary of Changes

The added documentation provides a comprehensive guide on using the Llamabot CLI's `serve` command to start an Ollama compatible server for a querybot. It details the usage of the command, including the necessary parameters and their descriptions, to facilitate users in setting up their querybot server efficiently.

## Details

- Introduction of the `serve` command in Llamabot CLI for creating an Ollama compatible server for querybots.
- Detailed usage instructions for the `serve` command, including an example command line.
- Explanation of key parameters for the `serve` command such as `--system-prompt`, `--collection-name`, `--document-paths`, `--model-name`, `--host`, and `--port`.
- The documentation aims to enable users to easily set up their querybot server by providing clear and concise instructions.